### ISSUE TITLE
Error after update to Matomo 5 and LoginLdap 5.0.3 

### ISSUE THREAD

**Issue by morenstrat** [piwik/plugin-LoginLdap#357]
Since the update I'm getting the following error during CRON execution:

```
INFO      [07:55:24] 2353556  ---------------------------
INFO      [07:55:24] 2353556  NOTES
INFO      [07:55:24] 2353556  - Async process archiving supported, using CliMulti.
INFO      [07:55:24] 2353556  - Reports for today will be processed at most every 900 seconds. You can change this value in Matomo UI > Settings > General Settings.
INFO      [07:55:24] 2353556  - Archiving was last executed without error 9 hours 45 min ago.
INFO      [07:55:24] 2353556  ---------------------------
INFO      [07:55:24] 2353556  START
INFO      [07:55:24] 2353556  Starting Matomo reports archiving...
INFO      [07:55:24] 2353556  1 out of 3 archivers running currently
INFO      [07:55:24] 2353556  Start processing archives for site 1.
INFO      [07:55:24] 2353556    Will invalidate archived reports for yesterday in site ID = 1's timezone (2023-12-19 00:00:00).
INFO      [07:55:25] 2353556  'DEBUG     [07:55:24] 2353598  UserSynchronizer::makeConfigured(): LDAP access synchronization not enabled.
DEBUG     [07:55:24] 2353598  UserSynchronizer::makeConfigured: configuring with defaultSitesWithViewAccess = all
DEBUG     [07:55:24] 2353598  UserSynchronizer::makeConfigured(): LDAP access synchronization not enabled.
DEBUG     [07:55:24] 2353598  UserSynchronizer::makeConfigured: configuring with defaultSitesWithViewAccess = all
{"idarchives":[576],"nb_visits":327}'
ERROR     [07:55:25] 2353556  Error unserializing the following response from ?module=API&method=CoreAdminHome.archiveReports&idSite=1&period=day&date=2023-12-19&format=json&trigger=archivephp: 'DEBUG     [07:55:24] 2353598  UserSynchronizer::makeConfigured(): LDAP access synchronization not enabled. DEBUG     [07:55:24] 2353598  UserSynchronizer::makeConfigured: configuring with defaultSitesWithViewAccess = all DEBUG     [07:55:24] 2353598  UserSynchronizer::makeConfigured(): LDAP access synchronization not enabled. DEBUG     [07:55:24] 2353598  UserSynchronizer::makeConfigured: configuring with defaultSitesWithViewAccess = all {"idarchives":[576],"nb_visits":327}'
INFO      [07:55:25] 2353556  Finished archiving for site 1, 1 API requests, Time elapsed: 0.297s [1 / 1 done]
INFO      [07:55:25] 2353556  Done archiving!
INFO      [07:55:25] 2353556  ---------------------------
INFO      [07:55:25] 2353556  SUMMARY
INFO      [07:55:25] 2353556  Processed 0 archives.
INFO      [07:55:25] 2353556  Total API requests: 1
INFO      [07:55:25] 2353556  done: 1 req, 332 ms, 1 errors.
INFO      [07:55:25] 2353556  Time elapsed: 0.332s
INFO      [07:55:25] 2353556  ---------------------------
INFO      [07:55:25] 2353556  SCHEDULED TASKS
INFO      [07:55:25] 2353556  Starting Scheduled tasks... 
INFO      [07:55:25] 2353556  done
INFO      [07:55:25] 2353556  ---------------------------
INFO      [07:55:25] 2353556  ---------------------------
INFO      [07:55:25] 2353556  SUMMARY OF ERRORS
INFO      [07:55:25] 2353556  Error: Error unserializing the following response from ?module=API&method=CoreAdminHome.archiveReports&idSite=1&period=day&date=2023-12-19&format=json&trigger=archivephp: 'DEBUG     [07:55:24] 2353598  UserSynchronizer::makeConfigured(): LDAP access synchronization not enabled. DEBUG     [07:55:24] 2353598  UserSynchronizer::makeConfigured: configuring with defaultSitesWithViewAccess = all DEBUG     [07:55:24] 2353598  UserSynchronizer::makeConfigured(): LDAP access synchronization not enabled. DEBUG     [07:55:24] 2353598  UserSynchronizer::makeConfigured: configuring with defaultSitesWithViewAccess = all {"idarchives":[576],"nb_visits":327}'
ERROR     [07:55:25] 2353556  1 total errors during this script execution, please investigate and try and fix these errors.
ERROR     [07:55:25] 2353556  Uncaught exception: /var/www/example.com/web/core/CronArchive.php(631): 1 total errors during this script execution, please investigate and try and fix these errors. [Query: , CLI mode: 1]

In CronArchive.php line 631:
                                                                                                 
  1 total errors during this script execution, please investigate and try and fix these errors.  
                                                                                                 

core:archive [--url URL] [--skip-idsites [SKIP-IDSITES]] [--skip-all-segments] [--force-idsites [FORCE-IDSITES]] [--skip-segments-today] [--force-periods [FORCE-PERIODS]] [--force-date-last-n [FORCE-DATE-LAST-N]] [--force-date-range [FORCE-DATE-RANGE]] [--force-idsegments FORCE-IDSEGMENTS] [--concurrent-requests-per-website [CONCURRENT-REQUESTS-PER-WEBSITE]] [--concurrent-archivers [CONCURRENT-ARCHIVERS]] [--max-websites-to-process MAX-WEBSITES-TO-PROCESS] [--max-archives-to-process MAX-ARCHIVES-TO-PROCESS] [--disable-scheduled-tasks] [--accept-invalid-ssl-certificate] [--php-cli-options [PHP-CLI-OPTIONS]] [--force-all-websites] [--force-report [FORCE-REPORT]]
```

**Comment by AltamashShaikh** [comment]
@morenstrat Will prepare a PR to fix this

**Comment by bikramj** [comment]
I should have reached here earlier but after spending few hours troubleshooting, commenting out those 2 logger lines in `UserSynchronizer.php` fixed the archive cron run for now.

**Comment by AltamashShaikh** [comment]
@bikramj Released v5.0.4 of the plugin to resolve this error
